config/example/config.txt

```diff
@@ -202,13 +202,19 @@ GUEST_BAN
 #SERVERURL server.net:port
 
 ## forum address
-# FORUMURL http://example.com
+FORUMURL http://example.com

```

Comment:
Probably shouldn't uncomment the `http://example.com` lines in the example config.